lib/src/main/java/io/ably/lib/transport/ConnectionManager.java

```diff
@@ -711,6 +725,25 @@ public void run() {
 		}
 	}
 
+	private void handleReauth() {
+		pendingReauth = false;
```

Comment:
I guess if the single attempt fails, then a few seconds later the token will actually expire, and there will be a new transport after getting a new token. So the existing retry/fallback/backoff logic associated with the reconnection attempt will do what we need better than retrying the preemptive in-transport reauth